### PR TITLE
DHFPROD-8556: GraphE2E-ExpandRecordsOnTable

### PR DESCRIPTION
**Pull request by csilva-ml**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests
- N/A Added to Release Wiki

### CODE REVIEW

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx [hunk 1]
```diff
@@ -178,11 +180,30 @@ describe("Group Nodes", () => {
...
 
+  it("Validate Expand records on table", () => {
+    cy.wait(1000)
+    cy.log("**Right click and select expand in the table**");
+    graphExplore.focusNode(ExploreGraphNodes.PRODUCT_GROUP);
+    graphExplore.getPositionsOfNodes(ExploreGraphNodes.PRODUCT_GROUP).then((nodePositions: any) => {
+      let orderCoordinates: any = nodePositions[ExploreGraphNodes.PRODUCT_GROUP];
+      const canvas = graphExplore.getGraphVisCanvas();
+     
+      // Right click and expand 3 records of the node
```
Comment (xnikhil08) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx:198]: Can you update the comment here ? It still says expand 3 records of the node.

**Review by xnikhil08 — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx (hunk 1 above)
Comment (csilva-ml) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx:198]: Good catch, thanks!

**Comment by xnikhil08** [comment]
@csilva-ml - Could you please rebase ?

File: marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx [hunk 2]
```diff
@@ -178,11 +180,30 @@ describe("Group Nodes", () => {
...
+      let orderCoordinates: any = nodePositions[ExploreGraphNodes.PRODUCT_GROUP];
+      const canvas = graphExplore.getGraphVisCanvas();
+     
+      // Right click and expand all records on a table
+      canvas.rightclick(orderCoordinates.x, orderCoordinates.y, {force: true}).then(()=>{
+        graphExplore.getViewRecordsInTable().should("be.visible").click();
+      })
+
+      cy.log("**Validate all the records appear in the table**")
+      tables.getTableRows().should("contain", "50").should("contain", "60").should("contain", "70").should("contain", "80").should("contain", "90");
```
Comment (aniamy) [inline marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx:204]: Can we add a should **not** contain?

Comment (csilva-ml) [reply marklogic-data-hub-central/ui/e2e/cypress/integration/explore/graphGroupNodes.spec.tsx]: Done!

**Review by aniamy — APPROVED** [review]

**Review by aniamy — DISMISSED** [review]

**Review by xnikhil08 — APPROVED** [review]

**Review by briantang — APPROVED** [review]
Looks good

**Comment by briantang** [comment]
@csilva-ml This looked ready to merge but failed on linting. If you could run lint fix on this, then it would be good to go.

**Review by aniamy — APPROVED** [review]

**Review by aniamy — APPROVED** [review]

**Review by briantang — APPROVED** [review]

**Pull request merged**